ossdbtoolsservice/disaster_recovery/disaster_recovery_service.py

```diff
@@ -119,8 +121,12 @@ def _perform_backup(connection_info: ConnectionInfo, params: BackupParams, task:
         return TaskResult(TaskStatus.FAILED, str(e))
     pg_dump_args = [pg_dump_location,
                     f'--file={params.backup_info.path}',
-                    f'--format={_BACKUP_FORMAT_MAP[params.backup_info.type]}',
-                    "--disable-dynamic-loading"]
+                    f'--format={_BACKUP_FORMAT_MAP[params.backup_info.type]}']
+
+    # Add additional options for Mac systems
+    if sys.platform == 'darwin':
+        pg_dump_args.insert(-1, '--disable-dynamic-loading')
```

Comment:
Just curious what is the significance of this parameter

Reply:
Dynamic loading was causing issue when running pg executables (pg dump and restore) in osx environment so added this param to prevent that